biblatex-iso690.tex

```diff
@@ -316,7 +409,7 @@ \subsection{Database guide}
 \paragraph{Article in a serial}\hfill\\
 
 \c{t03}
-\begin{verbatim}
+\begin{ltxexample}
 @ARTICLE{t03,
   author        =   {LYNCH, C.},
```

Comment:
I just noticed this. ALL CAPS in `.bib` files are almost always a bad idea.
```suggestion
   author        =   {Lynch, C.},
```
Are there more entries like this?